src/openfermion/transforms/_conversion.py

```diff
@@ -432,6 +432,99 @@ def _majorana_term_to_fermion_operator(term):
     return converted_term
 
 
+def get_majorana_operator(operator):
```

Comment:
Add type hint that you define in the doc-string here.  
`operator: Union[PolynomialTensor, DiagonalCoulombHamiltonian, FermionOperator],...`

and return type

`-> MajoranaOperator:`

Reply:
No other function in the submodule was using type hints, I needed to import Union from typing